src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.contrib.qaframework.page;
+
+
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class AttachmentsPage extends Page{
+	
+	private static final By ATTACHMENTS_LINK = By.className("icon-share-alt edit-action right");
```

Comment:
move this into the dashboard page within distro

Reply:
Have refactored the code base